vignettes/teal-data-reproducibility.Rmd

```diff
@@ -98,7 +98,7 @@ data <- within(teal_data(), {
   head(i)
 })
 cat(get_code(data)) # retrieve all code
-cat(get_code(data, datanames = "i")) # retrieve code for `i`
+cat(get_code(data, names = "i")) # retrieve code for `i`
```

Comment:
should we also prefix with `teal.code`? or use deprecated `teal.data` (with `datanames`) and allow a warning about the deprecation to be displayed?

```suggestion
cat(teal.code::get_code(data, names = "i")) # retrieve code for `i`
```

Reply:
Alrighty, so we better move to `teal.code::get_code(names` in the vignette